src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/product_v2/product.yml

```diff
@@ -1,3 +1,37 @@
+admin_products_v2_index:
+  path: /
+  methods: GET
+  defaults:
+    _controller: PrestaShopBundle:Admin/Sell/Catalog/Product/Product:index
+    _legacy_controller: AdminProducts
+
+admin_cart_rules_search_grid:
```

Comment:
From what I see you forgot to implement endpoint for search, set action for form etc.?

Reply:
Yeah you are were right search was not working correctly. Implemented it now.

Reply:
If it doesn't produce this ugly "no image" graphic then it's ok for me :P I don't see anything in Chrome so maybe it can stay this way :)